src/hist/basehist.py

```diff
@@ -401,20 +403,45 @@ def plot2d_full(
 
         return hist.plot.plot2d_full(self, ax_dict=ax_dict, **kwargs)
 
+    def plot_ratio(
+        self,
+        other: "Union[hist.BaseHist, Callable[[np.ndarray], np.ndarray], str]",
```

Comment:
This is quoted again so that I could add `hist.BaseHist` to the types.

Reply:
(though maybe I'm being dumb and there is an obvious way to use a class type inside of itself without getting a circular import error)

Reply:
Yes I was being dumb. This was already solved by Henry with

```
T = TypeVar("T", bound="BaseHist")
```

:+1: